dist/phila-vue-datafetch.js

```diff
@@ -5462,12 +5464,26 @@
   };
 
   DataManager.prototype.fetchData = function fetchData () {
+      var this$1 = this;
```

Comment:
Why does this need to exist? Are we changing the store later in the method? If so, can we just extract the data we need? e.g - 
```
const activeCondoFeatureId = this.store.state.activeCondo.featureId;
```

Reply:
Ah, I deleted this from the main file but didn't fix the dist folder.